### ISSUE TITLE
EmptyBlocksCountAsWindows Problem

### ISSUE THREAD

**Issue by D3nnis3n** [StrangeLoopGames/EcoIssues#9201]
Setting that setting to "false" is gamebreaking, as you would need to make a door to complete a room, but can only make one at a carpentry table which needs to be in an enclosed room. The only workaround is making a room, getting into it, placing the table, close the room fully (e.g. no doors) and then make the doors - which probably is not the way it's meant to be played.

**Comment by pamduncan** [comment]
Yeah - well it used to just count windows as windows, not doors  That is odd
